src/components/Heroes/index.jsx

```diff
@@ -104,10 +119,13 @@ class RequestLayer extends React.Component {
         matchCount3,
         matchCount2,
         matchCount1,
+        matchCountTurbo,
```

Comment:
I'd group the turbo props together for readability

Reply:
sounds good, I followed the initial approach but the suggested one it makes it better when options are more than 2.